src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -115,14 +115,9 @@ public static boolean isJobTracked(final String jobName) {
    * @return a boolean to signify if the jobName is or is not blacklisted.
    */
   public static boolean isJobBlacklisted(final String jobName) {
-    final String[] blacklist = DatadogUtilities.joblistStringtoArray( DatadogUtilities.getBlacklist() );
-    final String jobNameLowerCase = jobName.toLowerCase();
-
-    if (blacklist != null) {
-      return Arrays.asList(blacklist).contains(jobNameLowerCase);
-    }
+    final List<String> blacklist = DatadogUtilities.joblistStringtoList( DatadogUtilities.getBlacklist() );
 
-    return false;
+    return blacklist.contains(jobName.toLowerCase());
```

Comment:
Could you remove whitespace from `jobName` here as well as lowering? When we set the whitelist and blacklist, spaces are stripped out. This would look like 
`return blacklist.contains(jobName.replaceAll("\\s", "").toLowerCase())`